src/main/java/org/carlmontrobotics/subsystems/Drivetrain.java

```diff
@@ -105,8 +110,17 @@ public class Drivetrain extends SubsystemBase {
     private Timer simTimer = new Timer();
 
     public Drivetrain() {
-        // SmartDashboard.putNumber("set x", 0);
-        // SmartDashboard.putNumber("set y", 0);
+        SmartDashboard.putNumber("Pose Estimator set x", getPose().getTranslation().getX());
+        SmartDashboard.putNumber("Pose Estimator set y", getPose().getTranslation().getY());
+        SmartDashboard.putNumber("Pose Estimator set rotation", getPose().getRotation().getDegrees());
+        setPose(new Pose2d(SmartDashboard.getNumber("Pose Estimator set x", getPose().getTranslation().getX()),
+                SmartDashboard.getNumber("Pose Estimator set y", getPose().getTranslation().getY()),
+                Rotation2d.fromDegrees(SmartDashboard.getNumber("Pose Estimator set rotation",
+                        getPose().getRotation().getDegrees()))));
```

Comment:
this might call a nullpointerexception b/c the pose estimator hasn't been initialized yet and you are trying to set its pose